cli/tests/e2e/snapshots/test_ci/test_lockfile_parse_failure_reporting/complete.json

```diff
@@ -27,5 +17,18 @@
       }
     },
     "engine_requested": "OSS"
-  }
+  },
+  "dependency_parser_errors": [
+    {
+      "path": "Pipfile.lock",
+      "parser": "jsondoc",
+      "reason": "expected one of ['\"', '-?(0|[1-9][0-9]*)([.][0-9]+)?([eE][+-]?[0-9]+)?', '[', 'false', 'null', 'true', '{']",
+      "position": {
```

Comment:
Can you double check this against the "not using position" comment in the ATD PR?